### DISCUSSION TITLE
Select Component background color change in latest paste version

### DISCUSSION THREAD

**Discussion by vignesh-sivaprakasam** [twilio-labs/paste#2305] [Q&A] [answered]
Hi Team,
     In the latest paste-core version the background-color of Select component is `#f4f4f6` where as in the older paste-core version the background-color is `white`. 
     This background-color change gave us the feeling that the component is disabled.
     
Code sandbox links:
     **Latest**: https://codesandbox.io/s/paste-team-select-pmvhqf?file=/src/App.js
     **Older**: https://codesandbox.io/s/proud-water-m9u77c?file=/src/App.js
     
Screenshots:
     **Latest**:     
<img width="753" alt="Screenshot 2022-03-21 at 10 16 27 AM" src="https://user-images.githubusercontent.com/35305421/159206967-ee936bbe-19fc-4cc9-895e-077f2dcf9dfb.png">

    **Older**:   
<img width="762" alt="Screenshot 2022-03-21 at 10 16 35 AM" src="https://user-images.githubusercontent.com/35305421/159207002-6b962632-0451-4a9b-87c8-9a0ab7510fc2.png">

**Comment by nkrantz** [comment, marked as answer]
Hi @vignesh-sivaprakasam - thank you for posting this Discussion. It does look disabled with the color token update, this is an unexpected change that came from a recent fix. We're working on a solution now and will post in this thread when it's back!

**Reply by nkrantz** [reply]
The change went in to fix the colors and you'll be able to see the changes as soon as [the Version Packages PR](https://github.com/twilio-labs/paste/pull/2314) gets merged for the next release. Feel free to follow along. Thanks again!

**Reply by vignesh-sivaprakasam** [reply]
Thank you for the quick response @nkrantz .
